package/gluon-mmfd/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-mmfd
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-mmfd
+  TITLE:=mesh multicast forwarding daemon - gluon integration
```

Comment:
maybe one or another capital letter would be nice here, especially for "Gluon", maybe also "Mesh" as it's the beginning

Reply:
ok.